build.sbt

```diff
@@ -1,6 +1,19 @@
-name := "spark-web-model"
+lazy val root = project.in(file("."))
+  .settings(Seq(
+    name := "spark-web-model",
```

Comment:
All SBT settings are defined in a way of

```scala
setting := value
```

Where `setting` is predefined. All settings are available [here](https://www.scala-sbt.org/1.1.0/api/sbt/Keys$.html), but documentation is really poor.

You can read it as assignment to variable (there's slight difference, but not relevant)